tests/Go/Console/Command/DebugWeavingCommandTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Go\Console\Command;
+
+use Go\Functional\BaseFunctionalTest;
+
+class DebugWeavingCommandTest extends BaseFunctionalTest
```

Comment:
I'm trying to understand this test, how it's possible to weave differently each time?